packages/cli/src/commands/components/constants.ts

```diff
@@ -1,54 +1,11 @@
 import type { SpaceDatasource } from '../datasources/constants';
 
-export interface SpaceComponent {
-  name: string;
-  display_name: string;
-  created_at: string;
-  updated_at: string;
-  id: number;
-  schema: Record<string, any>;
-  image?: string;
-  preview_field?: string;
-  is_root?: boolean;
-  is_nestable?: boolean;
-  preview_tmpl?: string;
-  all_presets?: Record<string, unknown>;
-  preset_id?: number;
-  real_name?: string;
-  component_group_uuid?: string;
-  color: null;
-  internal_tags_list: SpaceComponentInternalTag[];
-  internal_tag_ids: string[];
-  content_type_asset_preview?: string;
-}
-
-export interface SpaceComponentGroup {
-  name: string;
-  id: number;
-  uuid: string;
-  parent_id: number | null;
-  parent_uuid: string | null;
-}
+import type { ComponentFolders, Components, InternalTags, Presets } from '@storyblok/management-api-client';
 
-export interface SpaceComponentPreset {
-  id: number;
-  name: string;
-  preset: Record<string, unknown>;
-  component_id: number;
-  space_id: number;
-  created_at: string;
-  updated_at: string;
-  image: string;
-  color: string;
-  icon: string;
-  description: string;
-}
-
-export interface SpaceComponentInternalTag {
-  id: number;
-  name: string;
-  object_type?: 'asset' | 'component';
-}
+export type SpaceComponent = Components.Component;
+export type SpaceComponentGroup = ComponentFolders.ComponentFolder;
```

Comment:
question(non-blocking): would it make sense to make the type names the same? Talking about either a Group or a Folder makes it confusing.

Reply:
Not sure if it's something in our shoes to fix, the API mixes those, because to the end user, they are folders, but in the API properties, they are called groups.